en/documents.html

```diff
@@ -99,8 +99,38 @@ <h3 id="id-scheme">id scheme</h3>
 </tbody>
 </table>
 
+<h3 id="docid-in-results">Document IDs in search results</h3>
+<p>
```

Comment:
link / refer to https://docs.vespa.ai/en/reference/default-result-format.html#id